### PR TITLE
5.40.0 Fixes - Batch 2 (`themes/src`)

### PR DESCRIPTION
**Pull request by adrians5j**
## Changes
This PR moves all of the code in the `extensions/theme` folder to `extensions/theme/src`. This was made for consistency (the new scaffolds also follow this folder structure).

With that, one additional change we did here is the import of the `global.scss` file. Previously, the file was directly imported from Admin/Website React apps, but now, they are imported within the the `theme` package's code. 

## How Has This Been Tested?
Manually.

## Documentation
Changelog.

### CODE REVIEW

File: packages/cwp-template-aws/template/ddb/dependencies.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "typescript": "4.7.4"
   },
   "workspaces": {
-    "packages": ["apps/admin", "apps/website", "apps/api/graphql", "extensions/theme", "packages/*"]
+    "packages": ["apps/admin", "apps/website", "apps/api/graphql", "packages/*", "extensions/theme"]
```
Comment (Pavel910) [inline packages/cwp-template-aws/template/ddb/dependencies.json:53]: It should be `extensions/*`, just like `packages/*`, and then you never need to modify these rules again.

Comment (adrians5j) [reply packages/cwp-template-aws/template/ddb/dependencies.json:53]: The thing is, users can also create extensions in e.g. `extensions/fm/api/xyz` folder.

Comment (Pavel910) [reply packages/cwp-template-aws/template/ddb/dependencies.json:53]: If you think everyone will be using scaffolds, then probably this is ok.

Comment (adrians5j) [reply packages/cwp-template-aws/template/ddb/dependencies.json:53]: Yes, every extension article we'll have will essentially start with scaffold run. Examples:

https://docs-webiny-com-git-release-5400-webiny.vercel.app/docs/file-manager/extending/customize-file-preview
https://docs-webiny-com-git-release-5400-webiny.vercel.app/docs/file-manager/extending/customize-file-actions

Tip: the Getting Started section is actually generated via [a React component](https://github.com/webiny/docs.webiny.com/pull/659/files#diff-5ea5a0a8cd8c4b551d0e1e9a1090329c96916375060cb64d5247db012f9b45b5).

**Comment by adrians5j** [comment]
Went through this changes with @Pavel910.

**Pull request merged**